File: engine/person.java
package engine;

import java.util.ArrayList;

public class person {
	
	String name = "duke nukem";
	int health = 100;
	int score = 0;
	ArrayList<Item> inventory = new ArrayList<Item>();
	
	public person(){}
	
	public person(String name, int health, int score){
		this.name = name;
		this.health = health;
		this.score = score;
		ArrayList<Item> inventory;
	}
	
	String getName(){
		return name;
	}

	int getHealth(){
		return health;
	}

	int getScore(){
		return score;
	}

	ArrayList<Item> getInventory(){
		return inventory;
	}

}

File: person.js
/*Do not change whats in this block of comments ---------------------------------------------------------------------*/
var myjavaclass = Java.type('script.personScript');

function person(name, health, score){
	this.name = name;
	this.health = health;
	this.score = score;
	
	this.getPerson = function(){
		var ArrayList = Java.type('java.util.ArrayList');
		var myPerson = new ArrayList();
		myPerson.add(this.name);
		myPerson.add(this.health);
		myPerson.add(this.score);
		return myPerson;
		}
}
/* -------------------------------------------------------------------------------------------------------------------*/


/* feel free to change or alter this for whatever you would like, after this line ---------------------------------------------------*/
/* just need to keep the same format to not have errors */

var person1 = new person("venom", "100", "0");
myjavaclass.person(person1);

var person2 = new person("spiderman", "65", "1000");
myjavaclass.person(person2);
